SU2_CFD/src/output/CFlowOutput.cpp

```diff
@@ -2402,6 +2402,14 @@ void CFlowOutput::WriteForcesBreakdown(const CConfig* config, const CSolver* flo
           file << "Laminar Viscosity (non-dim): " << config->GetMu_ConstantND() << "\n";
           break;
 
+        case VISCOSITYMODEL::COOLPROP:
+          file << "Viscosity Model: CoolProp  \n";
+          file << "Laminar Viscosity: " << "--";
+          if (si_units) file << " N.s/m^2.\n";
+          else file << " lbf.s/ft^2.\n";
+          file << "Laminar Viscosity (non-dim): " << config->GetMu_ConstantND() << "\n";
```

Comment:
If the model implies variable properties it is misleading to print constant values that are not used by the solver.
Unless I'm missing something?
```suggestion
```